torchvision/datasets/folder.py

```diff
@@ -32,20 +32,32 @@ def is_image_file(filename):
     return has_file_allowed_extension(filename, IMG_EXTENSIONS)
 
 
-def make_dataset(dir, class_to_idx, extensions):
+def make_dataset(dir, class_to_idx, extensions, is_valid_file=None):
```

Comment:
can you make `extensions=None` here? Because now it is not a required arg anymore

Reply:
@fmassa thanks for the reply and all your suggestions looks completely valid to me and  will update PR soon.